data/hai/hai reveal 5 philosophers.txt

```diff
@@ -0,0 +1,1075 @@
+			label "actual questions"
+			choice
+				`	"Do you want me to take you home?"`
+					goto home
+				`	"Do you know anything about the microtranceivers?"`
+					goto "microtranceivers"
+				`	"What can you tell me about this city?"`
+					goto place
+				`	"What do you do here?"`
+					goto do
+				`	"What do you know about this area of space?"`
+					goto space
+				`	"How did the Korath get here?"`
+					goto korath
+				`	"Can you speak to other Korath for me?"`
+					goto "translation services"
+				`	"Why were there Korath Raiders in this system?"`
+					goto korath
+				`	"I have no more questions."`
+			`	"It was a pleasure meeting you," says the human leader.`
+				goto leave
+			label korath
+			`	The Korath leader tells you, "A few years ago, three of our ships jumped into the Alcyone system to raid. Numerous pirate vessels left Stormhold and disabled all of our ships. A large human ship boarded our flagship. All we heard was screaming; the pirates used deadly gas to kill the entire crew."`
+			`	"You call it, 'nerve gas.' The death is excruciatingly painful to Korath. Not wanting that fate, the crews of the second and third Korath vessels surrendered. Now we're here, safe, and free. We are not scientists like the others here, but we help as we can." The other two leaders smile at him gratefully.`
+			# Apparently you're not the only one who can pack a Bactrian with
+			# nerve gas and hundreds of disposable teenage soldiers. The pirates
+			# disable or destroy Korath Raiders all the time. It seems some are
+			# adventurous enough to capture the ships.
+			`	"The pirates must defend this planet against a space-based lifeform that attacks in flocks. Our ships gave them a significant advantage. Fortunately we have one more left, to defend against any threats." He eyes the Republic operatives as he says that.`
+			`	The Korath leader continues, "I must ask you hold secret the presence of Korath here. If the other Exiles hear of this place, they may attack to retrieve us, or simply to pillage. We will remain here in hiding until Korath space is safe."`
+				goto questions
+			label "translation services"
+			`	The Korath leader looks worried, "No, we cannot. Few among us Korath know your language, and we must remain here in secret. Perhaps in time there will be a way, but not yet."`
+				goto questions
+			label "microtranceivers"
+			`	The Korath leader looks to the Hai who responds, "Is that how you were led back here?"`
+			`	"Yes," you say.`
+			`	The Hai leader continues, "That is excellent, that was the plan. Our overlords ordered us to make microtranceivers to track their prey. The microtranceivers were a combination of Hai medical microbots and what our Korath friends here could tell us. None of the Korath were scientists, but the principles of their science are part of their engineers' training. With their knowledge, we augmented the transceivers to transmit in hyperspace as well, hoping it would lead someone back here."`
```

Comment:
```suggestion
			`	The Hai leader continues, "That is excellent, that was the plan. Our overlords ordered us to make microtransceivers to track their prey. The microtransceivers were a combination of Hai medical microbots and what our Korath friends here could tell us. None of the Korath were scientists, but the principles of their science are part of their engineers' training. With their knowledge, we augmented the transceivers to transmit in hyperspace as well, hoping it would lead someone back here."`
```